README.md

```diff
@@ -1 +1,46 @@
-# todo
\ No newline at end of file
+# Building a "To-do list" website
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+Tasks to do:
+
+- Project 1: List structure.
+- Project 2: Interactive list.
+- Project 3: Add and remove items.
+
+![Screen Shot]()
```

Comment:
- [ ] Kindly fix this broken image for completion of the professional README. 😊

Reply:
Fixed